services/llmService.js

```
@@ -2,10 +2,10 @@ const OpenAI = require('openai');
 
 class LLMService {
   constructor() {
+    if (LLMService.instance) {
+        return LLMService.instance;
```

Biome (1.9.4)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor is ignored.

(lint/correctness/noConstructorReturn)